doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1834,6 +1839,18 @@ Tactic notations allow to customize the syntax of tactics. They have the followi
              underlying entry can be used: entry is either ``constr``, ``hyp``, ``integer``
              or ``int_or_var``.
 
+   .. note:: A *smart* qualified identifier is:
+
+             - either a qualified identifier (possibly with empty qualification)
```

Comment:
Let's avoid defining syntax in text where possible.  What part of the grammar does this correspond to?  Let's hyperlink to the syntax or show it nearby.

Reply:
I suspect that the grammar is not explicitly highlighted anywhere. It appears for instance at the end of the documentation for [`unfold`](https://coq.inria.fr/refman/proof-engine/tactics.html?highlight=unfold#coq:tacn.unfold).

Reply:
OK, maybe just add a todo in the .rst with the reference, we'll fix it later.

Reply:
@jfehrle It is `smart_qualid` (used to be `smart_global` which we recently renamed).

Reply:
OK, thanks. Then will use it.